public/index.js

```diff
@@ -0,0 +1,20 @@
+const items = querySelector(.'todos')
+
+cells.forEach((cell) => {
+
+})
+
+})
+
+
+const buttn = querySelector('.completed')
+
+buttn.addEventListener('click', () => {
+
+})
+
+
+const incompletes = querySelector('.mustDo')
+incomplete.addEventListener('click', () => {
```

Comment:
What is `incomplete`? I would imagine you want to attach the event listener to the document or window or is incomplete the name of an element?